pkg/api/testing/fuzzer.go

```diff
@@ -117,8 +117,9 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 			c.FuzzNoCustom(j) // fuzz self without calling this function again
 		},
 		func(j *api.DaemonControllerStatus, c fuzz.Continue) {
```

Comment:
Oh yeah, I didn't understand the details of fuzz testing. I'll talk to you about this later IRL. Do values get automatically fuzzed? Also, I'm not sure why we don't fuzz the Spec.